api/candlepin-api-spec.yaml

```diff
@@ -3158,6 +3158,58 @@ paths:
         default:
           $ref: '#/components/responses/default'
 
+    put:
+      description: |
+        Updates the specified using the non-null fields specified in the provided EnvironmentDTO.
+        Note that some fields of an environment cannot be changed without deleting and recreating
+        the environment, such as its ID, or organization (owner); while some other properties that
+        are mutable may not be changed by this operation, such as environment content overrides.
+
+        Ignored EnvironmentDTO fields:
+        - id
+        - owner
+        - environmentContent
```

Comment:
Should we also include the created and updated fields here?

Reply:
Yes; done.